### PR TITLE
fix: nav display issues

### PR DESCRIPTION
**Pull request by jimlind**
### 🔗 Linked issue

* https://energysage.atlassian.net/browse/CED-468
* https://energysage.atlassian.net/browse/CED-472
* https://energysage.atlassian.net/browse/CED-493
* https://energysage.atlassian.net/browse/CED-497

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [x] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Resolves several issues found in the bug bash as noted above.
* Adds the My Marketplace link to the logged in drop-down
* Tightens up spacing on headings in desktop menus
* Resolves default font weight issue between anchors and labels
* Adds emphasis to call to action links in desktop menus

### 🥼 Testing

* Tested in Chrome responsive devtools

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.
- [x] I have documented testing approach

### CODE REVIEW

File: es-bs-base/scss/_nav.scss [hunk 1]
```diff
@@ -179,9 +179,9 @@ $nav-hover-delay: 300ms;
     font-size: $font-size-75;
   }
 
-  // All links should have font-weight-normal (weight 400)
+  // All links and labels in the navigation should start at a font-weight-normal (weight 400)
   // TODO: This overrides styles for `eyebrow`
-  a {
+  a, label {
```
Comment (ericdouglaspratt) [inline es-bs-base/scss/_nav.scss:184]: Stylelint reports an issue here - the `label` after the comma just needs to be on the next line. You can confirm locally by running `make lint` in the `es-ds` directory.

**Review by ericdouglaspratt — CHANGES_REQUESTED** [review]
Just one small thing for CI to pass, otherwise looks good.

File: es-bs-base/scss/_nav.scss (hunk 1 above)
Comment (jimlind) [reply es-bs-base/scss/_nav.scss:184]: Cool. Thanks for the detail.
I was in a hurry and didn't even look to see the big red ❌ on CI runs.

**Review by ericdouglaspratt — APPROVED** [review]

**Review by mpleroux — APPROVED** [review]
Everything looks good on es-site 👍

**Pull request merged**